optaplanner-webexamples/src/test/java/org/optaplanner/webexamples/common/rest/service/AbstractClientArquillianTest.java

```diff
@@ -1,68 +0,0 @@
-/*
- * Copyright 2015 JBoss Inc
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.optaplanner.webexamples.common.rest.service;
-
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.List;
-
-import org.apache.commons.io.IOUtils;
-import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.spec.WebArchive;
-import org.jboss.shrinkwrap.resolver.api.maven.archive.importer.MavenImporter;
-import org.junit.runner.RunWith;
-
-import static org.junit.Assert.assertEquals;
-
-@RunWith(Arquillian.class)
-@RunAsClient
-public abstract class AbstractClientArquillianTest {
-
-    private static final String POM_DIRECTORY_NAME = "optaplanner-webexamples";
-
-    @Deployment(testable = false)
-    public static WebArchive createDeployment() {
-        File file = findPomFile();
-        return ShrinkWrap.create(MavenImporter.class)
-                .loadPomFromFile(file)
-                .importBuildOutput()
```

Comment:
The fact that we can get rid of this is very nice, as this has limitations in case we'd do GWT compilation etc.